build.gradle

```diff
@@ -132,6 +132,7 @@ subprojects {
     // See https://github.com/kelloggm/checkerframework-gradle-plugin/issues/85 (and the associated PR #86) for why this flag is needed
     suppressLombokWarnings = false
     checkers = [
+      'org.checkerframework.checker.fenum.FenumChecker',
```

Comment:
Possibly can be eventually moved to CI-only checkers, as with `IndexChecker` & others below

Reply:
Done on next PR